geth/notification/fcm_provider.go

```diff
@@ -0,0 +1,46 @@
+package notification
+
+import (
+	"github.com/NaySoftware/go-fcm"
+	"github.com/status-im/status-go/geth/notification/message"
+	"github.com/status-im/status-go/geth/common"
+)
+
+// FCMProvider represents messaging provider for notifications.
```

Comment:
Maybe we could be more descriptive here? For people who will open this code for the first time, FCM says nothing. Something like: `// FCMProvider represents Firebase Cloud Messaging provider for notifications.`